docs/about-filecoin/primer.md

```diff
@@ -0,0 +1,147 @@
+
+## The Filecoin ecosystem
+
+### [Filecoin integrations for web3 infrastructure](https://www.youtube.com/watch?v=Q0oe6i7d1u4)
+
+Filecoin presents many promising business opportunities. This session explores a few key Filecoin integration opportunities for pinning services and other businesses looking to integrate Filecoin.
+
+Length: 1 hour
+
+Supplemental Materials:
+
+- [What is an IPFS Pinning Service?](https://medium.com/pinata/what-is-an-ipfs-pinning-service-f6ed4cd7e475#:~:text=An%20IPFS%20pinning%20service%20is,running%20your%20own%20IPFS%20nodes.) (Pinata explainer)
+- [IPFS Docs: Persistence, permanence and pinning](https://docs.ipfs.io/concepts/persistence/)
+- [Filecoin Docs: Filecoin-backed pinning services](https://docs.filecoin.io/build/tools/filecoin-backed-pinning-services/)
+- [Developing on Filecoin](https://www.youtube.com/watch?v=aGCpq0Xf-w8) (video)
+
+### [Using Lotus to connect to the Filecoin network](https://www.youtube.com/watch?v=3dkzLJuv1jc&list=PLXzKMXK2aHh43-aCLLuGnny7nhnsf49LO&index=14&t=0s)
+
+Lotus, the reference implementation of the Filecoin protocol, is your path to connecting to the Filecoin network and interacting with your peers\! In this interactive workshop, we’ll go over the architecture of Lotus, walk through installing Lotus and syncing the blockchain, and examine how you can explore Filecoin’s functionality through the Lotus node.
+
+Length: 1 hour
+
+Supplemental Materials:
+
+- [Announcing lotus, our first alternate Filecoin implementation](https://filecoin.io/blog/announcing-lotus/)
+- [Filecoin Docs: lotus and go-ipfs interactions](https://docs.filecoin.io/build/examples/network-inspector/lotus-and-go-ipfs-interactions/)
+- [How to set up the lotus and go-ipfs API clients](https://docs.filecoin.io/build/examples/network-inspector/step-3-set-up-the-lotus-and-go-ipfs-api-clients/)
+
+### [Textile’s tools for the Filecoin network](https://www.youtube.com/watch?v=IZ8M9m9_uJY)
+
+Textile is a set of open-source tools that provide a decentralized database, IPFS-based storage, content hosting, and more, over libp2p, IPFS, and Filecoin. This workshop covers how to use Textiles Threads, Buckets, and Hub tools to build apps for the Filecoin network (focus on JavaScript). The session includes an introduction to Textile’s suites of tools, basic command-line (CLI) actions, and Textile’s JavaScript Client(s). To prepare, we encourage you to install a few things before we get started:
+
+- [Install](https://docs.textile.io/hub/accounts/) the Textile Hub command line interface (CLI) on your computer (so that you can connect to Textile’s Hub via the Hub CLI tools).
+- [Set up Node.js](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs) on your computer.
+
+Length: 1 hour 20 minutes
+
+Supplemental Materials:
+
+- [A protocol & event-sourced database for decentralized user-siloed data](https://blog.textile.io/introducing-textiles-threads-protocol/)
```

Comment:
Link to our example apps using Powergate is missing.

Also should link the Powergate page.

Reply:
Unless this is specifically about Hubs because Powergate is below, in that case section should be renamed. Otherwise maybe merged.